Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG RUST_VERSION=1.73-bookworm
+ARG VARNISH_VERSION=7.5
+
+# we need the same debian version on the rust and varnish so
+# that libssl-dev and libssl3 match
+FROM rust:${RUST_VERSION}
+
+WORKDIR /vmod_reqwest
+ARG VMOD_REQWEST_REPO=gquintard/vmod_reqwest
```

Comment:
I assume this should be `rers`?

Reply:
actually, why should we even have this file in the root?  We need the `docker/Dockerfile` for development, but I don't think we should be distributing vmod-specific docker image